### ISSUE TITLE
LLVM_CMAKE_PATH does not find LLVMConfig.cmake on Fedora

### ISSUE THREAD

**Issue by tdaff** [flang-compiler/flang#290]
I was trying to install using the system LLVM on Fedora, which seems to work fine, but I have to hack the `CMakeLists.txt` file in order to get it to search the correct path for the `LLVMConfig.cmake`:

```
 sed -i 's|  set(LLVM_CMAKE_PATH "${LLVM_BINARY_DIR}/lib/cmake/llvm"|  set(LLVM_CMAKE_PATH "${LLVM_BINARY_DIR}/lib64/cmake/llvm"|' ../CMakeLists.txt
```

Source: https://github.com/tdaff/flang-docker/blob/de387286f0f6c362620efc2923255ae2fd774354/Dockerfile

**Comment by junghans** [comment]
I think this was fixed by #88.

**Comment by tdaff** [comment]
Yes, it is, thanks!
